pom.xml

```diff
@@ -234,6 +234,17 @@
     </build>
 
     <profiles>
+        <profile>
+            <id>development</id>
```

Comment:
As a suggestion: the add-on is build and released with `clean install`. To run the demo locally we need `vaadin-core` dependency. 
Would it be enough to have only one development profile, which runs always if you execute just `mvn` and ignored when you want to build the add-on?
Why the prod mode needed? IMO it usually needed in the target project where the add-ons are used, not in the add-on itself. So I propose to remove it, if no objections.

Reply:
Good point Mikhail, good suggestions 👍 

Just to repeat if I understand correctly:
- remove `production` profile,
- make `development` the default one, even if not explicitly added (with `<activation><activeByDefault>true</activeByDefault></activation>`)
- add `jetty:run` build goald to `development` profile
- **Anything we need to do in build server, any config change? It will work without prod mode/build right?**

And with default profile men, we will be running properly the addon. Seems nice to me and probably for customers too.

I am not sure if this is correct, however, if this is the extent of the changes and it doesn't get more complicated, then I am happy to do the proposal! 🙇 

If it is a more complicated change, then I would be more happy to merge this as I am struggling to test my existing and new PR-s (as I need to add constantly the pom changes to run properly the jetty server)

So if it is more complicated then I would tackle the profile clean-up in a different PR.
(let's say if we cannot merge this by tomorrow)! 🙇

Reply:
> Just to repeat if I understand correctly:

> remove production profile,
> make development the default one, even if not explicitly added (with <activation>> <activeByDefault>true</activeByDefault></activation>)
>add jetty:run build goald to development profile

Yes, that was my point 👍 

> Anything we need to do in build server, any config change? It will work without prod mode/build right?

Currently, snapshot validation and release build is running with `clean install` without any profile, thus, I think this should be fine. One thing we should do when we will be ready to add IT tests is to add a production more to the "it" profile, so that the tests would run in prod mode as it usually should be.